extern/legacy_remove_me/.gitignore

```diff
@@ -1,3 +1,5 @@
 # override specific package dlls#
 ###################
 !bin/
+!*.exe
+!*.pdb
```

Comment:
did we really deliver .pdbs to our integrators?

Reply:
The bin-diff job highlighted the deletion of the pdb file so I thought it was safer to add it back as the job is run against release builds if I'm not mistaken.